app.rb

```diff
@@ -0,0 +1,28 @@
+require_relative 'lib/takeaway'
+require 'sinatra/base'
+require 'twilio-ruby'
+
+class App < Sinatra::Base
+  def takeaway(takeaway_class = TakeAway)
```

Comment:
there's no need to create methods that have injected dependencies inside a sinatra controller, but not in an action. Potentially you could investigate using a before filter: http://www.sinatrarb.com/intro.html if you want to.

Instantiate classes when needed in the controller action.